NEWS.md

```diff
@@ -1,6 +1,9 @@
 Hit tab to unselect buttons and scroll through the text using UP/DOWN or PGUP/PGDN.
 All announcements are stored in `/opt/openhabian/docs/CHANGELOG.md` for you to lookup.
 
+## new config option initialconfig ## May 8, 2021
+to automatically import an openHAB3 configuration from file or URL
```

Comment:
Please make this a complete separate sentence to help with clarity, and be sure to add this announcement to `docs/CHANGELOG.md`.

Reply:
ok